jsonrpc/debug_endpoint.go

```diff
@@ -273,6 +276,32 @@ func (d *Debug) GetRawHeader(filter BlockNumberOrHash) (interface{}, error) {
 	)
 }
 
+// GetRawTransaction returns the bytes of the transaction for the given hash.
+func (d *Debug) GetRawTransaction(txHash types.Hash) (interface{}, error) {
+	return d.throttling.AttemptRequest(
+		context.Background(),
+		func() (interface{}, error) {
+			tx, block := GetTxAndBlockByTxHash(txHash, d.store)
+
+			if tx == nil {
+				if txPending, ok := d.store.GetPendingTx(txHash); ok {
+					tx = txPending
+				}
+			}
+
+			if tx == nil {
+				return nil, fmt.Errorf("tx %s not found", txHash.String())
+			}
+
+			if block != nil && block.Number() == 0 {
+				return nil, ErrTraceGenesisBlock
+			}
```

Comment:
IMO we don't need this check here, because we are not tracing the block, but the transaction and genesis block does not contain any transactions by definition.

```suggestion
```